.github/workflows/spmd_tests.yaml

```diff
@@ -37,19 +37,22 @@ jobs:
           if [ -f spmd/requirements_dev.txt ]; then pip install -r spmd/requirements_dev.txt; fi
       - name: Test with pytest
         run: |
-          pytest --shard-id=${{ matrix.shard }} --num-shards=4 --cov=spmd test/spmd/
+          pytest --cov=spmd test/spmd/
+          # temporarily disable the pytest-shard logic
```

Comment:
nit: is "#" the correct semantic to comment out in yaml file? Looks like with indention it might not properly do the comment?